packages/flutter_tools/templates/ffi_plugin/src.tmpl/projectName.c.tmpl

```diff
@@ -0,0 +1,26 @@
+#include "{{projectName}}.h"
+
+// A very short-lived native function.
+//
+// For very short-lived functions, it is fine to call them on the main isolate.
+// They will block the Dart execution while running the native function, so
+// only do this for native functions which are guaranteed to be short-lived.
+MYLIB_EXPORT intptr_t sum(intptr_t a, intptr_t b) { return a + b; }
+
+// Platform independent macro for sleeping a thread.
```

Comment:
Let's not demonstrate things in the template that are strongly against Google C(++) style. There's no reason this can't be a function with ifdef'd implementation.